apps/utr-backend/src/endpoints/utils/AuthenticationUtils.ts

```diff
@@ -17,3 +33,35 @@ export async function isAuthenticated(req: any, res: any, next: any) {
 		return;
 	}
 }
+
+export async function getRole(req: any, res: any, next: any) {
```

Comment:
Anstatt in der Middleware nur die Rolle des Users zur Verfügung zu stellen, macht es, denke ich, mehr Sinn, das gesamte User-Objekt mitzugeben. Wenn wir die role im claim setzen, kann der folgende code sich daraus ja auch einfach die rolle ablesen.

Reply:
Okay, weil man dort dann auch noch den Namen z.B. auslesen kann oder wo wäre der Vorteil gegenüber nur der Rolle?

Danke für die Ausführlichen Antworten!

Reply:
Das gesamte User-Objekt wird jetzt mitgegeben. Die Rolle ist durch `req.user.role` auslesbar.